assets/js/base/components/radio-control/style.scss

```
@@ -2,14 +2,13 @@
 	@include reset-typography();
 	@include with-translucent-border( 0 0 1px );
 	display: block;
-	padding: $gap-small $gap-small 0 #{$gap-larger * 2};
+	padding: $gap 0 $gap #{$gap-larger + 4};
```

If I'm not wrong, this should be equivalent:
```suggestion
	padding: $gap 0 $gap $gap-largest;
```